newIDE/app/src/UI/ClosableTabs.js

```diff
@@ -192,6 +188,9 @@ export function ClosableTab({
                 id={id ? `${id}-button` : undefined}
                 {...longTouchForContextMenuProps}
                 focusRipple
+                // If touch ripple is not disabled, dragged preview will use the size of the ripple
+                // and it will be too big
```

Comment:
Uber nitpicking (good practice says that a comment ends by a period like a sentence):
```suggestion
                // If the touch ripple is not disabled, the dragged preview will use the size of the ripple
                // and it will be too big.
```